rpc/core/tx_status.go

```diff
@@ -0,0 +1,43 @@
+package core
```

Comment:
There is already a `tx.go` file. I think it makes sense to have this method there (next to the the transaction and txSearch methods)

Reply:
I'm putting it there but i couldn't find where those methods were tested so i kept the test file.